frontend/src/scenes/error-tracking/groups/OverviewTab.tsx

```diff
@@ -50,6 +57,27 @@ export const OverviewTab = (): JSX.Element => {
     )
 }
 
+const ViewSessionButton = ({ event }: { event: ErrorTrackingGroupEvent }): JSX.Element | null => {
+    const { openSessionPlayer } = useActions(sessionPlayerModalLogic)
+
+    const properties = JSON.parse(event.properties)
+
+    return (
+        <LemonButton
+            size="small"
+            onClick={() => {
+                const fiveSecondsBeforeEvent = dayjs(event.timestamp).valueOf() - 5000
+                openSessionPlayer({ id: properties.$session_id }, Math.max(fiveSecondsBeforeEvent, 0))
+            }}
+            disabledReason={
+                !properties.$session_id ? 'There was no $session_id associated with this exception' : undefined
```

Comment:
We should probably use the more descriptive name for `$session_id`, maybe read the descriptive name from the taxonomy?